syft/src/main/java/org/openmined/syft/domain/JobRepository.kt

```diff
@@ -199,6 +200,11 @@ internal class JobRepository(
                 }
                 .compose(config.networkingSchedulers.applySingleSchedulers())
     }
+
+    fun getLocalModelsPath(config: SyftConfiguration) = "${config.filesDir}/$jobId/models"
```

Comment:
Better to create a new class JobDataSource that stores jobId, config.filesdir, file paths and these functions. Feed this datasource to Kob repository